bip-0003.md

```diff
@@ -0,0 +1,709 @@
+<pre>
+  BIP: 3
+  Title: Updated BIP Process
+  Author: Murch <murch@murch.one>
```

Comment:
`<murch@murch.one>` doesn't show up when github displays the file. [BIP 348](https://github.com/bitcoin/bips/blob/master/bip-0348.md), [BIP 349](https://github.com/bitcoin/bips/blob/master/bip-0349.md) and [BIP 379](https://github.com/bitcoin/bips/blob/master/bip-0379.md) also have this problem. Making it a code block (` ``` ... ``` `) seems to work, otherwise replacing them with `&lt;` and `&gt;` would be an option if this section is kept as a `<pre>` block.

Reply:
Thank for pointing this out. Our formatting checker currently requires the tags `<pre>` and `</pre>` to encapsulate the preamble. That means that we would either need to have different formats for markdown and mediawiki, or change our formatting requirements and adjust all existing BIPs.

For the moment, I have opened #1759 to fix the unrendered email addresses in the existing markdown BIPs and fixed it here by using the HTML Entity Names as you proposed.

Reply:
Unfortunately, using the HTML Entity Names causes the formatting checker to fail, so this will need a more invasive fix. :-/
Still working on the approach.

Reply:
This is hopefully to be resolved by #1759, after which I’ll update here.